specification/httpx/src/main/scripts/org/kaazing/specification/httpx/extended/client.sends.message.between.opening.and.extended.handshake/request.rpt

```diff
@@ -15,7 +15,7 @@
 #
 
 property writeMask ${httpx:randomBytes(4)}

```

Comment:
I think this was a mistake.  We should use standard same ports and paths ("8080" and "path"), just like all the other specification tests.  @sanjay-saxena can you back out this change, and switch to 8080 and path